### PR TITLE
Fix wt2 os4

### PR DESCRIPTION
**Pull request by pb82**
Make WT2 compatible with OS4:

* instead of sharing one realm with other users, the user is instructed to create their own realm
* rewrite OS3 specific parts for OS4
* fix some status indicators (wrong service names)

Make WT5 compatible with OS4:

* instead of sharing one realm with other users, the user is instructed to create their own realm
* rewrite OS3 specific parts for OS4

### CODE REVIEW

File: walkthroughs/2-protecting-apps-sso/walkthrough.adoc [hunk 1]
```diff
@@ -108,8 +108,8 @@ To secure your application with single sign-on and allow additional users to log
 
 Follow these steps to create a new user and set that user's password.
 
-. Go to the link:{sso-realm-url}[{realm-display-name}, window="_blank"] realm, which is running on your {customer-sso-name} service.
-.. If prompted, enter the username `{shared-realm-username}` and password `{realm-password}`.
+. Go to the link:{sso-user-realm-url}[{realm-name}, window="_blank"] realm, which is running on your {customer-sso-name} service.
+.. If prompted, login with your credentials.
```
Comment (tiffanynolan) [inline walkthroughs/2-protecting-apps-sso/walkthrough.adoc]: If prompted, log in with your username and password.

File: walkthroughs/2-protecting-apps-sso/walkthrough.adoc [hunk 2]
```diff
@@ -158,8 +158,8 @@ The *{create-messages-app}* is run from a Node.js server, so the Node.js
 demonstrate how to include a configuration and enable the adapter.
 
 
-. Go to link:{sso-realm-url}[SSO Realm, window="_blank"].
-. Enter the username `{shared-realm-username}` and password `{realm-password}` if prompted.
+. Go to link:{sso-user-realm-url}[SSO Realm, window="_blank"].
+. Login with your credentials if prompted.
```
Comment (tiffanynolan) [inline walkthroughs/2-protecting-apps-sso/walkthrough.adoc]: If prompted, log in with your username and password.

File: walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc [hunk 3]
```diff
@@ -232,8 +236,8 @@ Verify that you followed each step in the procedure above.  If you are still hav
 
 Follow these steps to create a client.
 
-. Go to the link:{sso-realm-url}[{realm-display-name}, window="_blank"] realm, which is running on your {customer-sso-name} service.
-.. If prompted, enter the username `{shared-realm-username}` and password `{realm-password}`.
+. Go to the link:{sso-user-realm-url}[{realm-display-name}, window="_blank"] realm, which is running on your {customer-sso-name} service.
+.. If prompted, login with your credentials.
```
Comment (tiffanynolan) [inline walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc]: If prompted, log in with your username and password.

**Review by tiffanynolan — APPROVED** [review]
Overall, the PR looks good. I may need to do some additional clean-up in WT5 to make sure we're using the Bold style for button names, field names, etc, but that's just some minor things.

**Review by evanshortiss — APPROVED** [review]

**Pull request merged**